script.js

```diff
@@ -1 +1,123 @@
+const form = document.querySelector(".input-box"),
+    input=form.querySelector("input"),
+    waitList=document.querySelector(".wait-list"),
+    completeList=document.querySelector(".complete-list");
 
+const TODO_LS='toDos';
+
+
+
+function loadToDos(){
+    const loadedToDos=localStorage.getItem(TODO_LS);
+    if(loadedToDos!==null){
+        const parsedToDos=JSON.parse(loadedToDos);
+        parsedToDos.forEach(function(todo){
+            paintToDo(todo.text);
+        });
+    }
+}
+
+function saveToDos(){
+    localStorage.setItem(TODO_LS,JSON.stringify(toDos));
+}
+const toDos=[];
+function paintToDo(text){
+    const plusLi = document.createElement('li');
+    const delBtn = document.createElement('img');
+
+    //const img=document.createElement("img");
+    delBtn.src="img/bin.png";
+    //delBtn.innerHTML="$";
+
+    //delBtn.append(img);
+    delBtn.addEventListener("click", deleteToDo);
+    const newId=toDos.length+1;
+    //plusLi.innerHTML = input.value;
+    const inputText=document.createTextNode(input.value);
+    plusLi.id=newId;
+
+    const span = document.createElement("span");
```

Comment:
```suggestion
    const todoText = document.createElement("span");
```

가능한 한 변수명은 구체적이고 모호하지 않게 다는 게 중요한 것 같습니다!